### ISSUE TITLE
03-efsfilesystem.yaml not updated in S3 (still using python2.7 as runtime)

### ISSUE THREAD

**Issue by gemathus** [aws-samples/aws-refarch-moodle#34]
Line 533 of [03-efsfilesystem.yaml](https://github.com/aws-samples/aws-refarch-moodle/blob/master/templates/03-efsfilesystem.yaml) species Python 3.9 as runtime. This is correct. Nevertheless, the [S3 file referenced as the template url for the EFS File System](https://s3.amazonaws.com/aws-refarch/moodle/latest/templates/03-efsfilesystem.yaml) still uses python2.7.

This causes the stack to fail with the following message:

~~~~
Resource handler returned message: "The runtime parameter of python2.7 is no longer supported for creating or updating AWS Lambda functions. We recommend you use the new runtime (python3.9) while creating or updating functions. (Service: Lambda, Status Code: 400, Request ID: 79d0f135-d0c4-4fb0-a771-17da637c3774, Extended Request ID: null)" (RequestToken: 909c90cf-5c0a-849e-94ed-e7bae428f3b5, HandlerErrorCode: InvalidRequest)
~~~~

**Comment by gemathus** [comment]
I got it working by using `03-rds.yaml` and `03-efsfilesystem.yaml` exactly like shown in this repo, not the S3 file that is referenced in `00-master.yaml`.

I'll leave this open...

**Comment by gemathus** [comment]
UPDATE
All the successful stacks that I managed to launch failed during installation (even after setting the moodle admin user lastip in the database). 

I tried something new, tho, and got it working and correctly installed:

1. Cloned this repository
2. Uploaded all templates to an S3 bucket in my AWS account
3. Modified 00-master.yaml to reference my bucket files instead of `https://s3.amazonaws.com/aws-refarch/moodle/latest/templates/*`

This worked for me and everything is fine now (I just need to figure out how to migrate my current moodle site to this new architecture).

It would appear that the S3 files are pulling old versions... Let me know if this is of any help.
